index.js

```diff
@@ -80,13 +80,19 @@ const path = require('path');
 //. Indicates a JSON response. The first argument will be the response status
 //. code, and the second will be converted to JSON and sent as-is.
 //.
+//# Redirect :: (Number, String) -> Response a
+//.
+//. Indicates a redirection. The first argument will be the response status
+//. code, and the second will be the value of the Location header.
+//.
 //# Next :: a -> Response a
 //.
 //. Indicates that this middleware does not form a response. The supplied value
 //. will be assigned to `res.locals` and the next middleware will be called.
 const Response = daggy.taggedSum('Response', {
   Stream: ['code', 'mime', 'stream'],
   Json: ['code', 'value'],
+  Redirect: ['code', 'url'],
```

Comment:
Adding a new constructor to a strict sum type that is exposed to users is a breaking change, because the signature of the catamorphism changed.